its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -922,7 +912,7 @@ public void testAzureFunctions_WithWrongBaseDirectory_AnalysisSucceeds() throws
       // this might fail if Azure changes the drive
       assertThat(buildResult.getLogs()).contains("Using longest common projects path as a base directory: 'C:\\'");
     } else {
-      String temporaryFolderRoot = temp.getRoot().getParent();
+      String temporaryFolderRoot = temp.getRoot().getCanonicalFile().getParent();
```

Comment:
Something changed in the new orchestrator and now the canonical file path is used instead.